src/notation/view/notationpaintview.cpp

```diff
@@ -804,6 +804,11 @@ qreal NotationPaintView::height() const
     return QQuickPaintedItem::height();
 }
 
+QPoint NotationPaintView::canvasPos() const
```

Comment:
Here everything had been moved aweay fom Qt stuff like `QPointF` to just `PointF`